core/services/src/main/java/org/exoplatform/services/attachments/utils/EntityBuilder.java

```diff
@@ -125,14 +126,14 @@ public static final Attachment fromAttachmentNode(RepositoryService repositorySe
     String openUrl = getUrl(documentService, attachmentsPath);
     attachment.setOpenUrl(openUrl);
 
-    String attachmentsVersion = getStringProperty(attachmentNode, "exo:baseVersion");
+    String attachmentsVersion = getStringProperty(originalAttachmentNode, "exo:baseVersion");
     attachment.setVersion(attachmentsVersion);
 
     LinkedHashMap<String, String> previewBreadcrumb = new LinkedHashMap<>();
     try {
       previewBreadcrumb = documentService.getFilePreviewBreadCrumb(attachmentNode);
     } catch (Exception e) {
-      LOG.error("Error while getting file preview breadcrumb " + attachmentNode.getUUID(), e);
+      LOG.error("Error while getting file preview breadcrumb " + originalAttachmentNode.getUUID(), e);
```

Comment:
```suggestion
      LOG.error("Error while getting file preview breadcrumb " + (NodeImpl) originalAttachmentNode).getIdentifier(), e);
```

Reply:
@ahamdi I think that since the originalAttachmentNode will be original node not the symlink, it is always referencable so getUUID will be also good IMO